etcdmain/etcd.go

```diff
@@ -389,3 +393,19 @@ func identifyDataDirOrDie(dir string) dirType {
 	}
 	return dirEmpty
 }
+
+func setupLogging(cfg *config) {
+	capnslog.SetGlobalLogLevel(capnslog.INFO)
+	if cfg.debug {
+		capnslog.SetGlobalLogLevel(capnslog.DEBUG)
+	}
+	if cfg.logPkgs != "" {
+		repoLog := capnslog.MustRepoLogger("github.com/coreos/etcd")
+		settings, err := repoLog.ParseLogLevelConfig(cfg.logPkgs)
+		if err != nil {
+			log.Warningf("Couldn't parse log level string: %s, continuing with default levels", err.Error())
```

Comment:
Nah. `repoLogger` is the handle to the set of all loggers in the repository, where `log` is the single one (`etcdmain` in this case) and all Formatters know which package a line came from.

The upgrade path won't touch repoLogger -- merely creating a `var log = capnslog.NewPackageLogger("github.com/coreos/etcd", "$PACKAGENAME")` in place of importing `log`. Even then, everything should still work -- and individual log calls can change their level (to log.Fatal, or log.Warning, for example)